src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
According to the checklist, you should avoid using static methods or fields unless they are constants. Here, the `random` and `colorSupplier` instances should not be static. Instead, you can create them inside the method where they are used or make them instance variables without the static modifier.